app/helpers/assets_helper.rb

```diff
@@ -1,19 +1,36 @@
 module AssetsHelper
   def assets_links(level_of_help, smod_applicable)
-    first_links = {
+    links = {
       t("estimate_flow.assets.guidance.#{level_of_help}.text") => t("estimate_flow.assets.guidance.#{level_of_help}.link"),
       t("estimate_flow.assets.other_property_guidance.text") => t("estimate_flow.assets.other_property_guidance.#{level_of_help}.link"),
     }
-    links = if level_of_help == "controlled"
-              first_links
-            else
-              first_links.merge({
-                t("estimate_flow.assets.disregarded_capital_guidance.text") => t("estimate_flow.assets.disregarded_capital_guidance.certificated.link"),
-              })
-            end
+
+    if level_of_help == "certificated"
+      links[t("estimate_flow.assets.disregarded_capital_guidance.text")] = t("estimate_flow.assets.disregarded_capital_guidance.certificated.link")
+
+      if FeatureFlags.enabled?(:special_applicant_groups)
+        links[t("estimate_flow.assets.prisoner_guidance.text")] = t("estimate_flow.assets.prisoner_guidance.link")
+        links[t("estimate_flow.assets.bankrupt_guidance.text")] = t("estimate_flow.assets.bankrupt_guidance.link")
+      end
+    end
 
     return links unless smod_applicable
 
     links.merge({ t("generic.smod.guidance.text") => t("generic.smod.guidance.#{level_of_help}.link") })
   end
+
+  def partner_assets_links(check)
```

Comment:
Just a question, not a blocker.

Why would we not set `@check.level_of_help` in the '_partner_assets.html.slim' partial? That way we could get rid this line:

`translation_text = check.controlled? ? "controlled" : "certificated"`

Reply:
Again, not sure I follow. Are you suggesting that we pass in the level of help as an argument here instead of the whole check? Either way, we need _some_ way of knowing which translation text to use. Or are you suggesting we use `@check.level_of_help` instead of `translation_text`, since they end up as the same thing?

For the latter, it's a hangover from back when `level_of_help` could be nil (specifically when the level of help _page_ was feature flagged). So we could drop that now.

Reply:
That we use  `@check.level_of_help` instead of  `translation_text` - thanks for explaining why we're not following that now